lms/static/certificates/sass/_components.scss

```diff
@@ -92,39 +92,42 @@
 // message to user-centric banner
 .wrapper-banner-user {
   box-shadow: inset 0 rem(2) rem(2) 0 $black;
-  background: palette(grayscale, dark);
+  background: $gray-dark;
 }
 
 .banner-user {
   .message-title {
     color: $white;
-    font-weight: font-weight(semi-bold);
+    font-weight: 600;
+    margin-top: 0;
   }
 
   .message-copy {
     color: $white;
   }
 
   .message-actions .action {
+    @include button-outline-variant($gray-x-back, $gray-dark);
 
-    // STATE: hover, active, focus
-    &:hover,
-    &:active,
-    &:focus {
-      // customizations from UXPL
-      color: palette(grayscale, dark);
-    }
+    padding: 0.625rem;
+    font-size: rem(14);
+    transition: color 0.125s ease-in-out 0s, border-color 0.125s ease-in-out 0s, background 0.125s ease-in-out 0s, box-shadow 0.125s ease-in-out 0s;
+    border: 1px solid;
+    border-radius: 3px;
+    cursor: pointer;
+    font-family: $font-family-sans;
+    font-weight: 600;
 
     .icon {
-      @include margin-right(spacing-horizontal(x-small));
+      @include margin-right(map-get($spacing-horizontal, x-small));
 
-      font-size: font-size(large);
+      font-size: rem(18);
     }
 
     // CASE: icon display only
     &.icon-only {
       .action-label {
-        @extend %a11y-hide;
+        display: none;
```

Comment:
I don't think display none is equivalent to `%a11y-hide`:
https://github.com/edx/ux-pattern-library/blob/cfee8ee7d088f1067571e5a195f1c98198412cb3/pattern-library/sass/global/_helpers.scss#L56

Does it make sense to use the bootstrap `@include sr-only;` mixin here instead? https://getbootstrap.com/docs/4.5/utilities/screen-readers/